awx/main/utils/licensing.py

```diff
@@ -383,12 +384,24 @@ def validate(self):
             current_instances = Host.objects.active_count()
         else:
             current_instances = 0
+        license_date = int(attrs.get('license_date', 0) or 0)
+        if HostMetric:
+            automated_instances = HostMetric.objects.count()
```

Comment:
After doing the db partitioning work counting any tables makes me twitch a little. I don't love memoizing this value either. Computed fields have bitten us too many times in the past also. 

Don't have a better solution currently so this is probably fine. There are probably bigger fish to fry on the conf endpoint.